src/stravalib/tests/unit/test_model.py

```diff
@@ -30,16 +30,31 @@
     "model_class,raw,expected_value",
     (
         (
-            # Removing root
+            Activity,
```

Comment:
this is a bit redundant. but i did test for both start and end. 

we don't actually call the `.lat` and `.lon` attrs anywhere. do we need to? if we do that would need to be a separate test.

Reply:
These properties do not have any complicated business logic, so they do not need to be tested separately, IMO.